Project.toml

```
@@ -32,6 +32,7 @@ JuMP = "0.21"
 Makie = "0.9"
 MathProgBase = "0.7"
 MiniQhull = "0.1"
+ModelingToolkit = "0.4, 3.10"
```

```suggestion
ModelingToolkit = "1.1.3, 3.10"
```
Not sure if this helps but this is the last version for Julia v1.0.